### DISCUSSION TITLE
Additional Dynamic Variables

### DISCUSSION THREAD

**Discussion by abertay-university** [directus/directus#3914] [Feature Requests]
It will be great to have more dynamic variables such as:
$TODAY (2021-02-03 00:00:00)
$TOMORROW (2021-02-04 00:00:00)
$YESTERDAY (2021-02-02 00:00:00)

Or combine them into a function like PHP's strtotime(). For example:
$NOW('-1 day') or $NOW('day') - 1
$NOW('+1 week') or $NOW('week') + 1
$NOW('+1 month') or $NOW('month') + 1

**Comment by benhaynes** [comment]
Love these... would be great to compile a comprehensive list of possible variables for review. Thanks!

**Reply by sabinehoi** [reply]
This also reminds me of SQL's `NOW()`, maybe there's a way to get a richer set of functions by safely embedding SQL

**Comment by akiarostami** [comment]
I would love to add one more dynamic variable as well: CURRENT_PROJECT

This would be helpful to for managing user access in multi-tenancy.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
